server/apps/microservices/src/processors/thumbnail.processor.ts

```diff
@@ -62,7 +62,7 @@ export class ThumbnailGeneratorProcessor {
 
     const temp = asset.originalPath.split('/');
     const originalFilename = temp[temp.length - 1].split('.')[0];
-    const jpegThumbnailPath = resizePath + originalFilename + '.jpeg';
+    const jpegThumbnailPath = join(resizePath, '/', originalFilename, '.jpeg';
```

Comment:
From my understanding of join, you shouldn't need to add the / as it will automatically add the platform specific seperator inbetween each argument. I believe this should be something more like ```join(resizePath, `${originalFilename}.jpeg`)```

Reply:
const jpegThumbnailPath = join(resizePath, `${originalFilename}.jpeg`);

Is this correct

Reply:
Yes that I believe would do what we need here, the backticks allow you to substitute variables into the string so you can add the .jpeg afterwards without another /